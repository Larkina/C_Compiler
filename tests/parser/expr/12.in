a(0),
a(!b),
a(h)(7),
call();