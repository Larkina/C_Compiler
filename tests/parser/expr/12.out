( , )
	( args )
		( a )
		( 0 )
	( , )
		( args )
			( a )
			( ! )
		( , )
			( args )
				( args )
					( a )
					( h )
				( 7 )
			( args )
				( call )